WordPress/gutenberg
# Mobile - Fix Buttons block text input

`Gutenberg Mobile PR` -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/2665

Fixes https://github.com/WordPress/gutenberg/issues/25623

## Description
This PR fixes and regression introduced by [this PR](https://github.com/WordPress/gutenberg/pull/23034) where some props were missing in the `RichText` component for the `Buttons` block causing it impossible to type any text in it.

## How has this been tested?
- Open the app with metro running
- Add a Buttons block
- **Expect** to be able to type in texts inside the buttons

## Screenshots 
Before | After
-|-
<kbd><img src="https://user-images.githubusercontent.com/4885740/94158530-c99dff80-fe82-11ea-8e2c-2a85c19b5c6d.gif" width="250" /></kbd> | <kbd><img src="https://user-images.githubusercontent.com/4885740/94167052-539e9600-fe8c-11ea-913c-429b5e4a442e.gif" width="250" /></kbd> 

## Types of changes
Bug fix

## Checklist:
